english/yoga/yoga.md

```diff
@@ -9,6 +10,8 @@ Going through the poses, you feel your body warm up and you feel the energy flow
 
 Calm music starts to play quietly from all directions.
 
+Please relax. I can’t emphasise that enough. All of us experience stress, whether you’re a heart surgeon making vital incisions, or just Dave Bloggs queuing for a rail ticket behind a man who’s buying a travel pass, which involves photographs, scissors, forms being filled in, and his access won’t wipe. You get the picture. Stress like this just won’t go away, and it has to be combated.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.